src/hue_api.ts

```diff
@@ -47,3 +48,29 @@ export const request = (method, path, body) => new Promise((resolve, reject) =>
   req.write(JSON.stringify(body));
   req.end();
 });
+
+export const getRoomsFromGroups = (groups: Groups) => {
+  const rooms = [];
```

Comment:
Would be nice to type this more strongly:

```suggestion
  const rooms: Room[] = [];
```

and elsewhere
```
type Room = {
    id: string;
    name: string;
    ...;
}
```

Reply:
DONE!